cluster/engine_test.go

```diff
@@ -156,10 +158,12 @@ func TestEngineCpusMemory(t *testing.T) {
 	apiClient := engineapimock.NewMockClient()
 	apiClient.On("Info", mock.Anything).Return(mockInfo, nil)
 	apiClient.On("ServerVersion", mock.Anything).Return(mockVersion, nil)
+	apiClient.On("NetworkList", mock.Anything,
+		mock.AnythingOfType("NetworkListOptions"),
+	).Return([]types.NetworkResource{}, nil)
+	apiClient.On("VolumeList", mock.Anything, mock.Anything).Return(types.VolumesListResponse{}, nil)
```

Comment:
I'd rather that we be as specific as possible. Especially since all types are changing, and just in case any stray arguments get passed it'd be nice to know (even though for this particular case I don't see that happening). It also helps with debugging in the future.